src/views/Article/Article.js

```diff
@@ -83,7 +96,46 @@ export default class ArticleView extends PureComponent {
 					{children}
 				</Link>
 			);
-		} else {
+		}
+		else if (isGithubLink(href)) {
```

Comment:
whats this special link handling do/for?

Reply:
It transforms plain github links into:

```markdown
[vuetifyjs/vuetify](https://github.com/vuetifyjs/vuetify)
[FortAwesome/Font-Awesome#17097](https://github.com/FortAwesome/Font-Awesome/issues/17097)
[@dyc3](https://github.com/dyc3)
```

So they render like this without any extra work:

[vuetifyjs/vuetify](https://github.com/vuetifyjs/vuetify)
https://github.com/FortAwesome/Font-Awesome/issues/17097
[@dyc3](https://github.com/dyc3)